pylast/__init__.py

```diff
@@ -211,7 +211,8 @@ class _Network(object):
 
     def __init__(
             self, name, homepage, ws_server, api_key, api_secret, session_key,
-            submission_server, username, password_hash, domain_names, urls):
+            submission_server, username, password_hash, token, domain_names,
+            urls):
```

Comment:
Please put `token` at the end, in case anyone has been calling with positional parameters.